dataline-config/init/src/main/resources/config/SOURCE_CONNECTION_SPECIFICATION/37eb2ebf-0899-4b22-aba8-8537ec88b5a8.json

```diff
@@ -10,7 +10,8 @@
     "properties": {
       "start_date": {
         "type": "string",
-        "description": "Start getting data from that date."
+        "description": "Start getting data from that date.",
+        "examples": ["YYYY-MM-DD"]
```

Comment:
are examples also going to be shown on the UI?

Reply:
also, should we use a regex to define the date format here?

Reply:
> are examples also going to be shown on the UI?

yes. just added it to the update i posted in slack. luckily json schema supports this.

> also, should we use a regex to define the date format here?

yes. created an [issue](https://github.com/datalineio/dataline/issues/269). i think jsonschema supports this too actually. will handle separately from this PR.